### DISCUSSION TITLE
Swift Testing on examples

### DISCUSSION THREAD

**Discussion by JavierLaguna** [pointfreeco/swift-composable-architecture#3395] [General]
What plans are there to update the Examples (Case Studies, etc.) to Swift Testing?

**Comment by mbrandonw** [comment]
Hi @JavierLaguna, we don't have any immediate plans since we still support Xcode 15.x and we wouldn't want to lose test coverage on Swift 5.9-5.10. Converting them is straightforward though, and writing tests in Swift Testing versus XCTest look pretty much identical, so hopefully not too much is missing by not having that yet.

**Reply by JavierLaguna** [reply]
Thank you @mbrandonw !
